File: dayFour/javascript/After Break/TaskOne.js
        
    const  scanner = require("prompt-sync")();

                    let sum = 0;

                    for(let count = 1; count <=10; count++){

                   const score = Number(scanner("Enter Scores: "));

              

                      sum += score;

                }

                       console.log(sum);

File: dayFour/java/AfterBreak/TaskOne.java
import java.util.Scanner;

    public class  TaskOne{

        public static void main(String [] args){

           
                    Scanner input = new Scanner (System.in);
              

                  int sum = 0;

                   for(int count = 1; count < 10; count++){

                     

                         System.out.print("Enter Scores: ");
  int scores = input.nextInt();

                    sum += count;
                System.out.println(sum);
}

}



}
